src/app/clusters/wifi_network_diagnostics_server/wifi_network_diagnostics_server.cpp

```diff
@@ -67,6 +70,23 @@ CHIP_ERROR WiFiDiagosticsAttrAccess::ReadIfSupported(CHIP_ERROR (ConnectivityMan
     return aEncoder.Encode(data);
 }
 
+CHIP_ERROR WiFiDiagosticsAttrAccess::ReadWiFiBssId(AttributeValueEncoder & aEncoder)
+{
+    ByteSpan bssid;
+
+    if (ConnectivityMgr().GetWiFiBssId(bssid) == CHIP_NO_ERROR)
+    {
+        ChipLogProgress(Zcl, "Node is currently connected to Wi-Fi network with BSSID:");
+        ChipLogByteSpan(Zcl, bssid);
+    }
+    else
+    {
+        ChipLogProgress(Zcl, "Node is not currently connected.");
```

Comment:
This case should encode null, no?  As in, we should be working with a `Nullable<ByteSpan>` here or so.

Reply:
ByteSpan bssid is empty with 0 length by default. Is this not ok here

Reply:
So first-of, the cluster definition is not aligned with the actual spec in that this field should be marked nullable. So, we should fix that.

Then, if you do what I'm suggesting above (i.e to replace the instantiation of the type based on the generated type), it will automatically instantiate a nullable type, which will automatically have a default value of 'null', avoiding you needing to do anything special here.

Reply:
Acked, but I try to avoid using this type in platform layer APIs something like bellow to decouple the platform layer and app layer

CHIP_ERROR GetWiFiBssId(chip::app:Clusters::Attributes::Bssid::Type & value);**

Reply:
Nullable<ByteSpan> is not supported yet, add TODO to follow up after we get darwin converted over to the new APIs.